doc/en/source/k8sIntegration.rst

```diff
@@ -0,0 +1,186 @@
+
+- **site_id**: each FogFlow node (either cloud node or edge node) requires to have a unique string-based ID to identify itself in the system;
+- **physical_location**: the geo-location of the FogFlow node;
+- **worker.capacity**: it means the maximal number of docker containers that the FogFlow node can invoke;  
+
+
+Change values.yaml file
+---------------------------
+
+-Change config.json and nginx.conf path in values.yaml as per the environment.
+-Change externalIPs as per the environment.
+
+.. code-block:: console
+
+      # Default values for fogflow-chart.
+      # This is a YAML-formatted file.
+      # Declare variables to be passed into your templates.
+
+    ConfigMap:
+    data:
+      config.json: /home/necuser/fogflow/fogflow/yaml/heml/
+      nginx.conf: /home/necuser/fogflow/fogflow/yaml/heml/
+
+    serviceAccount:
+     # Specifies whether a service account should be created
+     create: true
+    # Annotations to add to the service account
+     annotations: {}
+    # The name of the service account to use.
+    # If not set and create is true, a name is generated using the fullname template
+      name: ""
+
+    service:
+      type: ClusterIP
+      port: 80
+
+    Service:
+     spec:
+      externalIPs:
+      - 172.30.48.24
+
+	  
+Start all Fogflow components with helm
+-------------------------------------------------------------
+
+Execute Helm command outside from FogFlow-chart location to start FogFlow Components.
+
+.. code-block:: console
+ 
+          helm install . --generate-name
+
+
+Validate the setup
+-------------------------------------------------------------
+
+There are two ways to check if the FogFlow cloud node is started correctly: 
+
+- Check all the Pods are Up and Running using "kubectl get pods --namespace=<namespace_name>"
+
+.. code-block:: console  
+
+         kubectl get pods --namespace=fogflow
+		 
+		 
+        NAME                           READY   STATUS              RESTARTS   AGE
+        cloud-broker-c78679dd8-gx5ds   1/1     Running             0          8s
+        cloud-worker-db94ff4f7-hwx72   1/1     Running             0          8s
+        designer-bf959f7b7-csjn5       1/1     Running             0          8s
+        dgraph-869f65597c-jrlqm        1/1     Running             0          8s
+        discovery-7566b87d8d-hhknd     1/1     Running             0          8s
+        master-86976888d5-drfz2        1/1     Running             0          8s
+        nginx-69ff8d45f-xmhmt          1/1     Running             0          8s
+        rabbitmq-85bf5f7d77-c74cd      1/1     Running             0          8s
+
+		
+- Check the system status from the FogFlow DashBoard
+
+System status can also be verified from FogFlow dashboard on web browser to see the current system status via the URL: http://<coreservice_ip>/index.html
+
```

Comment:
Add the limitation section for 
1. Worker thread implementation
2. Edge node K8s Support
3. Security and Network Policy
4. Taints and Trait
5. Performance Evaluation
6. Other Functionality